plugin.php

```diff
@@ -2,7 +2,7 @@
 
 /*
   Plugin Name: Gallerya
-  Version: 1.10.2
+  Version: 1.11.0
```

Comment:
As the scripts/styles identifiers are no longer the same, this is a backwards-incompatible change that may cause existing sites to no longer work correctly, so the major version should be bumped to 2.x.x.